src/db/z_db.erl

```diff
@@ -174,6 +180,7 @@ transaction_clear(Context) ->
 
 
 %% @doc Check if we have database connection
+-spec has_connection(#context{}) -> boolean().
```

Comment:
The spec in line 183 uses a record, please define a type for the record and use that instead.